openedx/features/learner_profile/views/learner_profile.py

```diff
@@ -42,10 +45,39 @@ def learner_profile(request, username):
         GET /account/profile
     """
     try:
-        return render_to_response(
+        context = learner_profile_context(request, username, request.user.is_staff)
+        # TODO: LEARNER-2554: 09/2017: Remove message and cookie logic when we no longer want this message
+        message_viewed = False
+        if (context['own_profile'] and
+                SHOW_PROFILE_MESSAGE.is_enabled() and
+                request.COOKIES.get('profile-message-viewed', '') != 'True'):
+            message_text = Text(_('Welcome to the new learner profile page. Your full profile now displays more '
+                                  'information to other learners. You can instead choose to display a limited '
+                                  'profile. {learn_more_link_start}Learn more{learn_more_link_end}')).format(
+                learn_more_link_start=HTML('<a href="http://edx.readthedocs.io/projects/open-edx-learner-guide/en/'
```

Comment:
I just realized that this needs to go through the help lookup system, so that it correctly picks up Open edX vs edX vs community documentation.